src/index.ts

```diff
@@ -44,7 +43,7 @@ export function convertTsConfig(
       sourceMaps: sourceMap,
       module: {
         type: ['commonjs', 'amd', 'umd'].includes(module) ? module : 'commonjs',
-        strict,
+        strict: false,
```

Comment:
i think we could just omit this field instead of giving it a default value though it's a correct one?

Reply:
Yep, we could. I set it to false as that was what was there before, but I'll take it out now.

Reply:
Updated the PR with this change